src/test/java/bio/terra/service/snapshotbuilder/query/QueryTestUtils.java

```diff
@@ -4,4 +4,9 @@ public class QueryTestUtils {
   public static TablePointer fromTableName(String tableName) {
     return TablePointer.fromTableName(tableName, s -> s);
   }
+
+  public static String collapseWhiteSpace(String string) {
```

Comment:
the goal of this is to help make testing sql more precise (instead of testing for substrings) and less fragile (instead of testing for the entire string including white spaces. It also allows changes of formatting to sql string to make it more readable without breaking tests.